config/locales/javascript/javascript.en.yml

```diff
@@ -239,7 +239,9 @@ en:
       contacts: "Contacts"
       posts: "Posts"
 
-    conversation:
+    conversations:
+      create:
+        no_recipient: "Hey, you need to add a recipient first!"
```

Comment:
This would mean that we would have to translate `javascripts.conversations.new.no_contacts` again, which was `javascripts.conversation.new.no_contacts` before. I don't think that renaming “conversation” to “conversations” is important enough to throw away the existing translations. That was the reason for the whole discussion about the translations and I still think that we should keep the old translations. (and thus keep the “conversation” key)